### ISSUE TITLE
Pagination component requires user to press enter after entering new page number

### ISSUE THREAD

**Issue by KevinFCormier** [patternfly/patternfly-react#6042]
**Describe the issue. What is the expected and unexpected behavior?**
If you type a new page number into the Pagination component, it does not take effect unless you press enter. This was counter-intuitive to me, and I expected just tabbing out of the number input would be sufficient. (Understandably, the page does not update immediately since the user may be typing a multi-digit number.)

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
enhancement

See related discussion here: https://patternfly.slack.com/archives/C293LQ36J/p1625084126080600
Also related to patternfly/patternfly#4224

**Comment by tlabaj** [comment]
cc @mcarrano @jessiehuff Can you provide input here about the expected interaction here

**Comment by mcarrano** [comment]
I agree with @KevinFCormier 's expectation, but I will defer to @jessiehuff here. Any reason that tabbing away from the field should not update the page?

**Comment by jessiehuff** [comment]
I think that the big question with this would be is it clear that as soon as they leave the field that they're going to leave the page? How would we clearly communicate to the user that the context is going to change? It's not very typical that an interaction would be activated by Tab as Tab is almost always navigation, except in the context of a form when you might expect your data to be saved as you move through the form with Tab. Would we update other input interaction patterns?

**Comment by KevinFCormier** [comment]
I'm fine to close this issue. Thanks @jessiehuff
